go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/google/nftables v0.2.1-0.20240414091927-5e242ec57806
 	github.com/google/uuid v1.6.0
 	github.com/goreleaser/nfpm/v2 v2.33.1
+	github.com/gorilla/websocket v1.5.0
```

Comment:
FWIW we elsewhere have mostly been using github.com/coder/websocket

Reply:
TIL, switched.